### ISSUE TITLE
Removing brush layers breaks scratchpad and gives errors

### ISSUE THREAD

**Issue by tinmanjuggernaut** [Zylann/godot_heightmap_plugin#295]
If you put two brushes on the Shapes list, then delete one, the scratchpad no longer works and you get these console errors:

```
SCRIPT ERROR: Invalid get index '1' (on base: 'Array').
          at: configure_paint_input (res://addons/zylann.hterrain/tools/brush/brush.gd:158)
```

Same index whether you delete the first or the second.

Godot 3.4.2
Win10/64
GTX1060
master

**Comment by Zylann** [comment]
I think I need to fix the index upon shape removal

**Comment by Zylann** [comment]
Fixed in e8cc8e047d5f862f5fdc7914e2bddc6c63a146e3

**Comment by tinmanjuggernaut** [comment]
Looks good, thanks.
